02-Reproject-Raster-In-R.Rmd

```diff
@@ -26,10 +26,11 @@ comments: false
 ##About
 
 It is common for us to encounter raster data layers that do not "line up". This 
-is most offen to do Coordinate Reference System (CRS) issues and can cause problems
+is most offen to do Coordinate Reference System (CRS) issues and can cause
```

Comment:
Edits (my text was messed up!):
Rasters that don't line up are most often in different Coordinate Reference Systems. This can cause problems with ...